libs/utils/WordPressUtils/src/test/java/org/wordpress/android/util/LogFileHelpersTest.kt

```diff
@@ -0,0 +1,64 @@
+package org.wordpress.android.util
+
+import android.content.Context
+import android.os.Build
+import androidx.test.core.app.ApplicationProvider
+import java.io.File
+import java.util.UUID
+import org.junit.After
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import org.wordpress.android.util.helpers.logfile.LogFileProvider
+
+@RunWith(RobolectricTestRunner::class)
+@Config(sdk = [Build.VERSION_CODES.O_MR1])
+class LogFileHelpersTest {
+    lateinit var testProvider: LogFileProvider
```

Comment:
Similar to my previous comment, I think we can make this a `private val` instead and get rid of the whole `setup` function:

```
private val testProvider = LogFileProvider.fromContext(ApplicationProvider.getApplicationContext())
```

Reply:
Changed in 1645984007f4980309c8bf45e9b6faba80c54d81